.changeset/long-swans-drive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/cli': minor
+---
+
+Add prompt in `warp init` command to choose if a trusted relayer should be used instead of making the choice by default for the user adn enable the `--yes` flag to default to a trusted ISM
```

Comment:
```suggestion
Add prompt in `warp init` command to choose if a trusted relayer should be used instead of making the choice by default for the user and enable the `--yes` flag to default to a trusted ISM
```